src/debugSession.ts

```diff
@@ -346,6 +361,8 @@ export class JanusDebugSession extends DebugSession {
         const connection = new DebugConnection(socket);
         this.connection = connection;
 
+        this.logVersionNumber();
```

Comment:
Logically that call should be part of the event handler. But it requires that the object this.connection is initialized and that is not done when the handler is called. 
(line 218 or 362)